frontend/js/main.js

```diff
@@ -0,0 +1,2203 @@
+}
+
+/**
+ * Function to store session info
+ *
+ * @param {string} driver database driver
+ * @param {string} path file path
+ * @param {string} fileName file name
+ * @param {string} sourceDb source db name
+ * @return {null}
+ */
+const storeResumeSessionId = (driver, path, fileName, sourceDb) => {
+  localStorage.setItem('driver', driver);
+  localStorage.setItem('path', path);
+  localStorage.setItem('fileName', fileName);
+  localStorage.setItem('sourceDb', sourceDb);
+}
+
+/**
+ * Function to read file content when clicked on resume session
+ *
+ * @param {string} driver database driver
+ * @param {string} path file path
+ * @param {string} fileName file name
+ * @param {string} sourceDb source db name
+ * @param {string} windowEvent hashchange or load event
+ * @return {null}
+ */
+const resumeSession = (driver, path, fileName, sourceDb, windowEvent) => {
+  filePath = './' + fileName;
+  readTextFile(filePath, async function (text) {
+    var data = JSON.parse(text);
+    localStorage.setItem('conversionReportContent', JSON.stringify(data));
+    sourceTableFlag = sourceDb;
+    fileData = await fetch(apiUrl + '/session/resume', {
+      method: 'POST',
+      headers: {
+        'Accept': 'application/json',
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({
+        "driver": driver,
+        "path": path,
+        "fileName": fileName
+      })
+    })
+    .then(function (response) {
+      if (response.ok) {
+        return response;
+      }
+      else {
+        return Promise.reject(response);
+      }
+    })
+    .catch(function (err) {
+      showSnackbar(err, ' redBg');
+    });
+    await ddlSummaryAndConversionApiCall();
+    await getInterleaveInfo();
+    jQuery('#importSchemaModal').modal('hide');
+    const { component = ErrorComponent } = findComponentByPath('/schema-report-resume-session', routes) || {};
+    document.getElementById('app').innerHTML = component.render();
+    showSchemaConversionReportContent();
+    if (windowEvent == 'hashchange') {
+      showSnackbar('schema resumed successfully', ' greenBg');
+    }
+  });
+}
+
+/**
+ * Callback function to read file content
+ *
+ * @param {file}
+ * @return {null}
+ */
+const readTextFile = (file, callback) => {
+  var rawFile = new XMLHttpRequest();
+  rawFile.overrideMimeType("application/json");
+  rawFile.open("GET", file, true);
+  rawFile.onreadystatechange = function () {
+    if (rawFile.readyState === 4 && rawFile.status == "200") {
+      callback(rawFile.responseText);
+    }
+  }
+  rawFile.send(null);
+}
+
+/**
+ * Function to create session table content
+ *
+ * @return {null}
+ */
+const setSessionTableContent = () => {
+  sessionArray = JSON.parse(sessionStorage.getItem('sessionStorage'));
+  if (sessionArray == null) {
+    document.getElementById('session-table-content').innerHTML = `<tr>
+      <td colspan='5' class='center session-image'><img src='assets/icons/Group 2154.svg' alt='nothing to show'></td>
```

Comment:
Where is the assets folder?

Reply:
It was removed before creating PR but I forgot to update this code . It is now updated

Reply:
ok